containers/dotnet-postgres/README.md

```diff
@@ -0,0 +1,153 @@
+# C# (.NET) and MS SQL
+
+## Summary
+
+*Develop C# and .NET Core based applications. Includes all needed SDKs, extensions, dependencies and a PostgreSQL container for parallel database development. Adds an additional PostgreSQL container to the C# (.NET Core) container definition.*
+
+| Metadata | Value |  
+|----------|-------|
+| *Contributors* | The VS Code Team, [Berkays](https://github.com/Berkays)  |
+| *Categories* | Core, Languages |
+| *Definition type* | Docker Compose |
+| *Published image architecture(s)* | x86-64 |
+| *Available image variants* | 3.1-focal, 5.0-focal, 6.0-focal |
```

Comment:
Does this work with the `-bullseye` definitions as well?  If so, you can change this as follows so you don't have to keep updating as new image variations appear:
```suggestion
| *Available image variants* | [See C# (.NET) definition](../dotnet). |
```